tests/integration/json_test.go

```diff
@@ -74,6 +74,16 @@ var _ = Describe("odojsonoutput", func() {
 			desiredDesCompJSON := `{"kind":"Component","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"nodejs","creationTimestamp":null},"spec":{"type":"nodejs","app": "myapp","source":"https://github.com/openshift/nodejs-ex"},"status":{"state":"Pushed"}}`
 			Expect(desiredDesCompJSON).Should(MatchJSON(actualDesCompJSON))
 
+			// odo storage create -o json
+			actualJSONStorage := helper.CmdShouldPass("odo", "storage", "create", "mystorage", "--path=/opt/app-root/src/storage/", "--size=1Gi", "-o", "json")
+			desiredJSONStorage := `{"kind":"storage","apiVersion":"odo.openshift.io/v1alpha1","metadata":{"name":"mystorage","creationTimestamp":null},"spec":{"size":"1Gi"},"status":{"path":"/opt/app-root/src/storage/"}}`
+			Expect(desiredJSONStorage).Should(MatchJSON(actualJSONStorage))
+
+			// odo storage list -o json
```

Comment:
Please remove the storage list scenario.
Reason - https://github.com/openshift/odo/pull/1828/files#r302371163